locators/payment_methods_locators.py

```diff
@@ -0,0 +1,35 @@
+from attrdict import AttrDict
+from selenium.webdriver.common.by import By
+
+
+class PaymentMethodsLocators:
+    _locators = AttrDict({
```

Comment:
Would this be better as an enum or a @dataclass so we could access them more like PaymentMethodsLocations.merchant_name etc?

Reply:
This approach was used in the base pgs python framework and I reused it in our framework but I can change it to enum or a @DataClass as you suggest :)

Reply:
I think enums won't work actually because we're setting these to tuples instead of just strings
 
But @dataclass is a newer way to do this rather than using AttrDict - came in python 3.6 I think. Out of interest what version of python are we using here?

Reply:
ok, I will look how @DataClass works and try replace to it
In our framework we are using python v3.7